golem-api-grpc/proto/golem/apidefinition/api_definition.proto

```diff
@@ -45,8 +58,21 @@ enum HttpMethod {
 }
 
 message WorkerBinding {
-  golem.component.ComponentId component = 1;
+  golem.component.VersionedComponentId component = 1;
```

Comment:
just small detail, we should rename component -> component_id  (to match model naming)